index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>My Notes Application</title>
+    <link rel="stylesheet" href="sanitize.css">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <h1>My Notes Application</h1>
+    </header>
+    <main>
+        <section class="main-section">
+            <div class="form-container" style="margin-bottom: 20px;">
+                <form id="note-form">
+                    <h2>Create a New Note</h2>
+                    <label for="note-title" class="form_label">Title:</label>
+                    <textarea id="note-title" placeholder="Enter note title"></textarea>
+                    <label for="note-content" class="form-label">Content:</label>
+                    <textarea id="note-content" placeholder="Enter note content"></textarea>
+                    <button type="submit" class="form_button">Add Note</button>
+                </form>
+            </div>
+            <div class="resource-container">
+                <h2>Resources</h2>
+                <ul>
```

Comment:
`<a>` elements contained in this list are not properly working.

Fix `href` attribute so user can navigate to expected url.

BTW, which is `blank` attribute responsibility?

Reply:
corrected code